easybuild/easyconfigs/p/psycopg2/psycopg2-2.9.5-GCCcore-11.2.0.eb

```
@@ -0,0 +1,30 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'psycopg2'
+version = '2.9.5'
+
+homepage = 'https://psycopg.org/'
+description = "Psycopg is the most popular PostgreSQL adapter for the Python programming language."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['a5246d2e683a972e2187a8714b5c2cf8156c064629f9a9b1a873c1730d9e245a']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('PostgreSQL', '13.4'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
```

should we add a path sanity check, or is that redundant with `sanity_pip_check`?
```suggestion

sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s'],
}

```